example/hands-on/services/bucky_hands_on/pc/pageobject/github_search_list.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.